src/main/java/com/bookstore/bookstoreapi/data/entity/Book.java

```diff
@@ -75,7 +77,7 @@ public class Book {
   @ToString.Exclude
   @OneToMany(mappedBy = "book", cascade = CascadeType.ALL)
   @Builder.Default
-  private Set<Comment> comments = new HashSet<>();
+  private List<Comment> comments = new ArrayList<>();
```

Comment:
Why is this a list now?

Reply:
Changed it to a list for readability in the JSON file. Nothing changes in terms of functionality, but it's just ordered better. (Ordered based on date and time comment was posted)